### PR TITLE
Created products page, added  methods to cart page and an appropriate…

### PR DESCRIPTION
**Pull request by ohalapats**
## Code reviewers

- [x] @1OtaMan1
- [ ] @ohalapats
- [ ] @oli4ka4321
- [ ] @schcurnazar
- [ ] @sibirtsv
- [ ] @SteTaras
- [ ] @wwwooo
- [ ] @yuriy-vantukh

### Second Level Review

- [ ] @demoxana
- [x] @lhalam

## Summary of issue
Created products page, added  methods to cart page and an appropriate locators, were written tests to cover cart functionality.

### CODE REVIEW

File: locators/cart.py [hunk 1]
```diff
@@ -18,3 +18,7 @@ class CartPageLocators(BasePageLocators):
         By.XPATH, '//*[@id="content"]/form/div/table/tbody/tr/td[4]/div/span/button[1]')
     BTN_DELETE = (
         By.XPATH, '//*[@id="content"]/form/div/table/tbody/tr/td[4]/div/span/button[2]')
+
+    BTN_DELETE_PRODUCT = (By.CLASS_NAME, 'btn btn-danger')
```
Comment (1OtaMan1) [inline locators/cart.py]: Good locators. Later we should think about 2+ items in the cart...

File: pages/cart.py [hunk 2]
```diff
@@ -37,7 +38,7 @@ def is_on_cart_page(self):
         """
         TODO
         """
-        if self.driver.current_url == "https://demo.opencart.com/index.php?route=checkout/cart":
+        if self.driver.current_url == "http://127.0.0.1/opencart.com/index.php?route=checkout/cart":
```
Comment (1OtaMan1) [inline pages/cart.py]: I suppose this string will change with every pull request.

Comment (ohalapats) [reply pages/cart.py]: Until we will deploy on a normal server

File: pages/home.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def is_on_home_page(self):
         """
         Make sure we on Home Page.
         """
-        if self.driver.current_url == "https://demo.opencart.com/":
+        if self.driver.current_url == "http://127.0.0.1/opencart.com/":
```
Comment (lhalam) [inline pages/home.py]: use urllib.parse

File: tests/guest/test_cart_as_guest.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
+from selenium import webdriver
+
+from pages.cart import CartPage
+from pages.home import HomePage
+
+driver = webdriver.Chrome()
+driver.get("http://127.0.0.1/opencart.com")
```
Comment (lhalam) [inline tests/guest/test_cart_as_guest.py]: use base_root_path

**Pull request merged**